frontend/test/__support__/e2e/helpers/e2e-qa-databases-helpers.js

```diff
@@ -148,6 +148,7 @@ export function restoreActionsDB() {
       INSERT INTO test_table (name) VALUES ('John'), ('Jane'), ('Jack'), ('Jill'), ('Jenny');
     `,
   });
+  cy.task("resetActionsDb");
```

Comment:
this task takes about 6 seconds on my machine. without the sample (orders/products/people, etc) dataset, it's more like 2.